packages/components/src/components/primitives/Dropdown/types.ts

```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+import { MenuButtonProps, MenuGroupProps, MenuItemProps, MenuListProps, MenuProps } from "@chakra-ui/react";
+
+export {
+    MenuProps as DropdownProps,
+    MenuButtonProps as DropdownButtonProps,
+    MenuItemProps as DropdownItemProps,
+    MenuListProps as DropdownListProps,
+    MenuGroupProps as DropdownGroupProps,
+} from '@chakra-ui/react';
+export interface DropdownStaticMembers {
+    Menu: FC<MenuProps>;
+    Group: FC<MenuGroupProps>;
+    Button: FC<MenuButtonProps>;
+    List: FC<MenuListProps>;
+    Item: FC<MenuItemProps>;
```

Comment:
please add the MenuDivider, MenuOptionGroup and MenuItemOption too

Reply:
Done